libs/model/src/policies/Nominations.policy.ts

```diff
@@ -0,0 +1,106 @@
+import { config, logger, Policy } from '@hicommonwealth/core';
+import { events } from '@hicommonwealth/schemas';
+import { ZodUndefined } from 'zod';
+import { models } from '../database';
+import { mustExist } from '../middleware';
+
+const log = logger(import.meta);
+
+const inputs = {
+  NominatorSettled: events.NominatorSettled,
+  NominatorNominated: events.NominatorNominated,
+  JudgeNominated: events.JudgeNominated,
+};
+
+export function NominationsWorker(): Policy<typeof inputs, ZodUndefined> {
```

Comment:
I have converted a few policies like this to a simpler switch invoking internal commands encapsulating validation/mutations... wondering if this will continue developing in complexity to justify another conversion